barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.test.cpp

```diff
@@ -29,6 +30,7 @@ namespace {
 auto& engine = numeric::get_debug_randomness();
 }
 
+STANDARD_TESTING_TAGS
```

Comment:
will you add a TODO to have more instant death tests? :)

Reply:
I'll add one more instant death test in the initialization. Instant death is a property of Origin Tags merging, we don't need to test for it in every endpoint if we test that the tags are getting merged